x-pack/plugins/maps/public/angular/get_initial_layers.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import _ from 'lodash';
+// Import each layer type, even those not used, to init in registry
+
+import '../layers/sources/wms_source';
```

Comment:
Why does this file import all of the sources from layers. Looks like those imports were added as part of https://github.com/elastic/kibana/pull/63297. Are they needed?

Reply:
Unfortunately yes. Because layer types are registered on import, we were running into the issue with embeddables, specifically the sample data dashboards, where it was trying to load unregistered layer types. I'm certain there's a more elegant solution but it likely needs to be a part of a separate PR that handles layer type registration in a slightly different way